test/src/e2e_vm_tests/harness.rs

```diff
@@ -64,7 +70,12 @@ pub(crate) async fn runs_on_node(
         signing_key: Some(SecretKey::from_str(SECRET_KEY).unwrap()),
         ..Default::default()
     };
-    run(command).await
+    run(command).await.map(|ran_scripts| {
+        ran_scripts
+            .first()
+            .map(|ran_script| ran_script.receipts.clone())
+            .unwrap()
```

Comment:
Could avoid the clone here with:
```rust
ran_scripts
    .into_iter()
    .next()
    .map(|ran_script| ran_script.receipts)
    .unwrap()
```

Reply:
I will address this right away since I found a bug when trying to deploy just a single member of the workspace, so will address them together